src/Components/MonacoWrapper.tsx

```diff
@@ -125,7 +151,10 @@ export const Monaco = ({
       width={width}
       options={{
         automaticLayout: true,
-        fontSize: 16,
+        fontSize: 12,
+        scrollBeyondLastLine: false,
```

Comment:
I think scroll beyond last line is helpful espcially for diff-editor etc. but don't mind trying this one out too. lets see

Reply:
Fair point, maybe enable a overwrite prop for over there?